src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -163,6 +175,48 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
     }
   }
 
+  private onSubmitCompleted = orderOrError => {
```

Comment:
#superminor #nonblocking `public` and `private` don't really matter with react component classes, since actual instances are never really exposed to anybody. There's a lot of other fields marked `private` in this class, so really super nonblocking.